nautobot/project-static/src/scss/nautobot.scss

```diff
@@ -834,6 +833,14 @@ body {
       position: absolute;
       width: $sidenav-link-segment-icon-width;
     }
+
+    &.flat {
```

Comment:
Wondering if we should establish a prefixed naming convention for Nautobot-specific CSS classes (as opposed to those that have meaning in Bootstrap, etc.) both for clarity and for avoiding collisions down the road?

At a minimum, feels like this should maybe be something like `sidenav-flat`?